docs/i18n/cn/docusaurus-plugin-content-docs/current/quick_start/install/operator.md

```diff
@@ -28,11 +28,11 @@ Hwameistor-Operator 负责自动化安装并管理 HwameiStor 系统。
    ```console
    helm repo add hwameistor-operator https://hwameistor.io/hwameistor-operator
    helm repo update hwameistor-operator
-   helm install hwameistor-operator hwameistor-operator/hwameistor-operator
+   helm install -n hwameistor hwameistor-operator hwameistor-operator/hwameistor-operator
    ```
 
 2. 创建 HwameiStor 存储系统
 
    ```console
-   kubectl apply -f https://raw.githubusercontent.com/hwameistor/hwameistor-operator/main/config/samples/hwameistor.io_hmcluster.yaml
+   kubectl -n hwameistor apply -f https://raw.githubusercontent.com/hwameistor/hwameistor-operator/main/config/samples/hwameistor.io_hmcluster.yaml
```

Comment:
tips: in latest operator version(v0.9.2), there is no need to apply `hwameistor.io_hmcluster.yaml` manually.

@buffalo1024  we should update this part according to latest operator usage.